########################################
# Messages
########################################
uint8 front_right
uint8 front_left
uint8 side_right
uint8 side_left
